diff --git a/node_modules/@girs/gobject-2.0/gobject-2.0.d.ts b/node_modules/@girs/gobject-2.0/gobject-2.0.d.ts
index 8207421..49145a0 100644
--- a/node_modules/@girs/gobject-2.0/gobject-2.0.d.ts
+++ b/node_modules/@girs/gobject-2.0/gobject-2.0.d.ts
@@ -3327,6 +3327,7 @@ export namespace GObject {
         static $gtype: GType<Object>;
 
         // Constructors
+        constructor(properties?: Partial<Object.ConstructorProps>, ...args: any[]);
 
         _init(...args: any[]): void;
 
