diff --git a/node_modules/@girs/gtk-4.0/gtk-4.0.d.ts b/node_modules/@girs/gtk-4.0/gtk-4.0.d.ts
index 954305d..93a7a7e 100644
--- a/node_modules/@girs/gtk-4.0/gtk-4.0.d.ts
+++ b/node_modules/@girs/gtk-4.0/gtk-4.0.d.ts
@@ -22995,7 +22995,7 @@ export namespace Gtk {
         /**
          * The currently focused window of the application.
          */
-        get activeWindow(): Window;
+        get activeWindow(): Window | null;
         /**
          * The menu model to be used for the application's menu bar.
          */
